tests/Jellyfin.Server.Implementations.Tests/Sorting/AiredEpisodeOrderComparerTests.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Emby.Server.Implementations.Sorting;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using Xunit;
+
+namespace Jellyfin.Server.Implementations.Tests.Sorting
+{
+    public class AiredEpisodeOrderComparerTests
+    {
+        [Theory]
+        [ClassData(typeof(EpisodeTestData))]
+        public void AiredEpisodeOrderCompareTest(BaseItem x, BaseItem y, int expected, bool err)
+        {
+            var cmp = new AiredEpisodeOrderComparer();
+            if (err == true)
+            {
+                Assert.Throws<ArgumentNullException>(() => cmp.Compare(x, y));
+            }
+            else
+            {
+                Assert.Equal(expected, cmp.Compare(x, y));
+                if (expected == 1)
```

Comment:
Why the if? Isn't it always the case that `Compare(x, y) == -cmp.Compare(y, x)`?

Reply:
If the result is 0 then switching x and y provides no added benefit. I added this to not waste time checking something that is not required. I can remove it if desired.

Reply:
Sorry I should not reply until after I drink my coffee in the morning.
Yes it should always be the case that `Compare(x, y) == -cmp.Compare(y, x)`
My goal here was to have as much code coverage as possible. Inside of compare there is logic to determine if x or y is a episode or special.

https://github.com/jellyfin/jellyfin/blob/804614315819c8c17c576b2cdfb683177b59eb7a/Emby.Server.Implementations/Sorting/AiredEpisodeOrderComparer.cs#L77-L82

If someone forgot to write the code on lines 77 to 80, then the tests may not catch that however if we only do compare(x,y) and do not try the reverse.

I did not try the reverse of 0 cases where 0 was returned because it felt unnecessary.

Reply:
Don't see the point of the if statement, just always do the check, you won't notice a difference in test execution speed

Reply:
Sorry guess I missed this one, not very used to the github PR review process and find it hard to follow comments.

Reply:
There's a fairly new feature on GitHub that might help you
![image](https://user-images.githubusercontent.com/675118/129523942-11a0524b-184c-4385-8cbe-53a2f0b484fc.png)